charts/datree-admission-webhook/templates/_scan-job.tpl

```diff
@@ -63,15 +63,22 @@ spec:
             seccompProfile:
               type: RuntimeDefault
           image: "{{ .Values.scanJob.image.repository }}:{{ .Values.scanJob.image.tag }}"
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
           resources: {{- toYaml .Values.scanJob.resources | nindent 12 }}
           volumeMounts:
             - name: webhook-config
               mountPath: /config
               readOnly: true
+            - name: custom-skip-list
+              mountPath: /config/datreeSkipList
+              readOnly: true
       volumes:
```

Comment:
what about the comment you added in the deployment? `# lagacy support for old configmap name`?